modules/data.land/R/put.veg.module.R

```diff
@@ -49,24 +35,29 @@
     logger.severe(paste(fcn, "does not exist."))
   }
   
+  # NOTE : match_pft is now called in veg2model
```

Comment:
the thing with both `match_species_id` and `match_pft` functions is that they require the veg data itself (or a particular column of it) to operate on. If we want them to be called outside the functions that write files, then we either need to pass the veg_info around (which we were trying to avoid I believe) or need to write a file after each step and pass around the file ids, then it would look like this:


- raw veg file
- load data & write: a somewhat standard veg file (like met CF files)
- match spp ids & write : veg data with matched spp ids (currently what is being written)
- match pfts & write : this file would also be model specific which makes it sort of unnecessary to write
- veg2model : final ic files (currently being written)

do we want to write all these steps, or am I missing something?